Makefile

```diff
@@ -25,8 +25,11 @@ firehose-writer:
 secret-detector:
 	mvn package -DfinalName=haystack-pipes-secret-detector -pl secret-detector -am
 
+span-key-extractor:
+	mvn package -DfinalName=span-key-extractor -pl span-key-extractor -am
+
 # build all and release
-release: clean json-transformer kafka-producer http-poster firehose-writer secret-detector
+release: clean json-transformer kafka-producer http-poster firehose-writer secret-detector span-key-extractor
```

Comment:
verify whether we need `span-key-extractor` it here. Since, it should be build as a dependency with other modules